ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -261,37 +261,62 @@ Get Model Inference
     [Documentation]    Returns the inference result after sending a POST request to a deployed
     ...    model endpoint. If token authentication is needed for the model, ${token_auth} should be
     ...    set to ${TRUE}.
-    [Arguments]    ${model_name}    ${inference_input}    ${token_auth}=${FALSE}
-    ...    ${project_title}=${NONE}
-    ${self_managed}=    Is RHODS Self-Managed
-    ${url}=    Get Model Route Via UI    ${model_name}
-    ${kserve}=    Run Keyword And Return Status    SeleniumLibrary.Page Should Contain    Single-model serving enabled
-    ${curl_cmd}=     Set Variable    curl -s ${url} -d ${inference_input}
-    IF    ${token_auth}
-        IF    "${project_title}" == "${NONE}"
-            ${project_title}=    Get Model Project    ${model_name}
+    [Arguments]    ${model_name}    ${inference_input}    ${token_auth}=${FALSE}   ${project_title}=${NONE}
+    ...    ${kserve_mode}=Serverless   ${deployment_mode}='UI'   ${service_port}=8888   ${end_point}=${NONE}
+    ${curl_cmd}=     Set Variable    ${NONE}
+
+    IF  ${deployment_mode} == 'UI'
+        ${self_managed}=    Is RHODS Self-Managed
+        ${url}=    Get Model Route Via UI    ${model_name}
+        ${kserve}=    Run Keyword And Return Status    SeleniumLibrary.Page Should Contain    Single-model serving enabled
+        ${curl_cmd}=     Set Variable    curl -s ${url} -d ${inference_input}
+        IF    ${token_auth}
+            IF    "${project_title}" == "${NONE}"
+                ${project_title}=    Get Model Project    ${model_name}
+            END
+            ${token}=    Get Access Token via UI    ${project_title}
+            ${curl_cmd}=     Catenate    ${curl_cmd}    -H "Authorization: Bearer ${token}"
+        END
+        IF  ${kserve}
+            Fetch Knative CA Certificate    filename=openshift_ca_istio_knative.crt
+            ${curl_cmd}=     Catenate    ${curl_cmd}    --cacert openshift_ca_istio_knative.crt
+        ELSE IF  ${self_managed}
+            Fetch Openshift CA Bundle
+            ${curl_cmd}=     Catenate    ${curl_cmd}    --cacert openshift_ca.crt
         END
-        ${token}=    Get Access Token Via UI    ${project_title}
-        ${curl_cmd}=     Catenate    ${curl_cmd}    -H "Authorization: Bearer ${token}"
     END
-    IF  ${kserve}
+    IF  ${deployment_mode} == 'Cli'
```

Comment:
For the CLI case you're forcing the Knative CA, but what if we want to have a CLI test case for modelmesh in a SM environment? You should add the openshift CA as well based on the conditions in 280/283